indexing-service/src/main/java/org/apache/druid/indexing/common/task/HadoopTask.java

```diff
@@ -173,15 +173,13 @@ public static ClassLoader buildClassLoader(final List<String> hadoopDependencyCo
       localClassLoaderURLs.addAll(Arrays.asList(hadoopLoader.getURLs()));
     }
 
-    ClassLoader parent = null;
-    if (JvmUtils.isIsJava9Compatible()) {
-      try {
-        // See also https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-A868D0B9-026F-4D46-B979-901834343F9E
-        parent = (ClassLoader) ClassLoader.class.getMethod("getPlatformClassLoader").invoke(null);
-      }
-      catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
-        throw new RuntimeException(e);
-      }
+    ClassLoader parent;
+    try {
+      // See also https://docs.oracle.com/en/java/javase/11/migrate/index.html#JSMIG-GUID-A868D0B9-026F-4D46-B979-901834343F9E
+      parent = (ClassLoader) ClassLoader.class.getMethod("getPlatformClassLoader").invoke(null);
```

Comment:
no need to use reflection anymore

Reply:
Have made the change.
As discussed offline, the animal-sniffer check was causing failures since it's currently configured to use JDK8 signature, and upgrading animal-sniffer can be a separate follow-up item after this.